preprocessors/action-recognition/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG PYTORCH="1.11.0"
+ARG CUDA="11.3"
+ARG CUDNN="8"
+FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
+
+# NVIDIA APT KEYS
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub
```

Comment:
Why do you need to the nvidia keys?

Reply:
I don't entirely understand I think it fulfills some lower level requirements but I can't build the container without it.

Reply:
Odd...

Reply:
I was using another image before that didn't have this requirement but it made the container pretty large so I switched to this one that I copied from semantic segmentation but I can switch it back or look for other options.